README.md

```diff
@@ -46,23 +46,26 @@ Receptor is hosted on AWS at *(In progress)*
          ```
          brew services start postgresql
          ```
-
+     4. To get Python working with Postgres, install psycopg2
+          ```
+          pip install psycopg2
+          ```
     </details>
 
 5. Create the receptor database
     1. Open the psql shell
         ```
         psql postgres
         ```
-    2. Create 'receptor' databse
+    2. Create 'receptor' databse. 
```

Comment:
Can you fix the spelling error in database?

Reply:
I missed this. I'll make the correction.